CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.11.0 (21.09.2023)
+
+- [Bump soroban-sdk to v20.0.0-rc2](https://github.com/kommitters/chaincerts-smart-contracts/issues/123)
```

Comment:
We must maintain consistency in the Changelog texts. For this case it should be: 

`Upgrade Soroban SDK to Preview 11`